modules/openy_gc_auth/modules/openy_gc_auth_custom/config/install/openy_gc_auth.provider.custom.yml

```diff
@@ -4,3 +4,4 @@ require_email_verification: 0
 email_verification_link_life_time: '14400'
 email_verification_text: 'Hello! <br> You’re just one step away from accessing your Virtual YMCA. Please open the link below to begin enjoying YMCA content made exclusively for members like you.'
 verification_message: 'We have sent a verification link to the email address you provided. Please open this link and activate your account. If you do not receive an email, please try again or contact us at XXX-XXX-XXXX to ensure we have the correct email on file for your membership.'
+one_time_link_no_valid_message: 'You have tried to use a one-time login link that has either been used or is no longer valid. Please request a new one using the form below.'
```

Comment:
Let's rename to `one_time_link_invalid_message`

Reply:
And please add an upgrade path for this param

Reply:
@hamrant thanks for review!
Fixed. Please re-check it.